application/org.openjdk.jmc.ui/src/main/java/org/openjdk/jmc/ui/misc/DndToolkit.java

```diff
@@ -112,8 +112,12 @@ public void dragFinished(DragSourceEvent event) {
 	}
 
 	public static <T, U> ViewerDropAdapter createLocalDropListTarget(
-		Viewer viewer, Class<T> targetType, Class<U> srcType, IDropAction<List<? extends U>, T> action,
-		IDropValidator<List<? extends U>, T> validator) {
+		Viewer viewer,
+		Class<T> targetType,
+		Class<U> srcType,
+		IDropAction<List<U>, T> action,
+		IDropValidator<List<U>, T> validator
+	) {
```

Comment:
**question:** It seems this change is not accepted by the formatter, I can change it back, but do you think we could improve readability if the formatter rules were updated.

```
Error:  Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.17.2:check (default-cli) on project org.openjdk.jmc.ui: The following files had format violations:
Error:      src/main/java/org/openjdk/jmc/ui/misc/DndToolkit.java
Error:          @@ -112,12 +112,8 @@
Error:           \t}
Error:           
Error:           \tpublic·static·<T,·U>·ViewerDropAdapter·createLocalDropListTarget(
Error:          -\t\tViewer·viewer,
Error:          -\t\tClass<T>·targetType,
Error:          -\t\tClass<U>·srcType,
Error:          -\t\tIDropAction<List<U>,·T>·action,
Error:          -\t\tIDropValidator<List<U>,·T>·validator
Error:          -\t)·{
Error:          +\t\tViewer·viewer,·Class<T>·targetType,·Class<U>·srcType,·IDropAction<List<U>,·T>·action,
Error:          +\t\tIDropValidator<List<U>,·T>·validator)·{
Error:           \t\treturn·createLocalDropTarget(viewer,·targetType,·new·IDropAction<ISelection,·T>()·{
Error:           
Error:           \t\t\t@SuppressWarnings("unchecked")
Error:  Run 'mvn spotless:apply' to fix these violations.
Error:  -> [Help 1]
```

Reply:
After `spotless:apply` the code becomes less readable in my opinion:

```diff
	public static <T, U> ViewerDropAdapter createLocalDropListTarget(
- 		Viewer viewer,
- 		Class<T> targetType,
- 		Class<U> srcType,
- 		IDropAction<List<U>, T> action,
- 		IDropValidator<List<U>, T> validator
- 	) {
+ 		Viewer viewer, Class<T> targetType, Class<U> srcType, IDropAction<List<U>, T> action,
+ 		IDropValidator<List<U>, T> validator) {
```